client/src/ui/atoms/form/expanding-textarea.tsx

```diff
@@ -10,8 +10,8 @@ export default function ExpandingTextarea(
 
   const resizeCallback = useCallback(
     (node: HTMLTextAreaElement) => {
-      if (value && node && node.scrollHeight + 2 > node.clientHeight) {
-        node.style.height = `${node.scrollHeight + 2}px`;
+      if (value && node && node.scrollHeight > node.clientHeight) {
```

Comment:
This bit is right: `scrollHeight` and `clientHeight` both include padding but not borders, so the box now expands at the correct point